configtools.spec

```diff
@@ -4,11 +4,16 @@ Release:     1
 Summary:     The configtools module
 
 License:     GPLv3+
-URL:         http://pypi.python.org/pypi/%{name}
-Source0:     http://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
+URL:         https://pypi.python.org/pypi/%{name}
+Source0:     https://pypi.python.org/packages/source/c/%{name}/%{name}-%{version}.tar.gz
 Buildarch:   noarch
 
+%if 0%{?rhel} == 7
 BuildRequires:	python2-devel
```

Comment:
Aren't we all in on Python3 now?

Reply:
Yes - I wasn't sure that RHEL7 could do it, but I did some testing and it seems to work properly (as long as we can get python3 and python3-devel RPMs which are in the RHEL7-latest repos).